src/main/java/KBot.java

```diff
@@ -0,0 +1,134 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+
+class Task {
+    private String description;
+    private boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public void markAsDone() {
+        this.isDone = true;
+    }
+
+    public void markAsNotDone() {
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "[X]" : "[ ]");
+    }
+
+    @Override
+    public String toString() {
+        return getStatusIcon() + " " + description;
+    }
+}
```

Comment:
Perhaps you can create a separate Java file called Task.java instead of writing all codes within your KBot.java file?